charts/kyverno-policies/templates/pod-sec-ctxt-vcs.yaml

```diff
@@ -0,0 +1,39 @@
+{{- $name := "pod-sec-ctxt-vcs" }}
```

Comment:
@rbak-hpe you can test how the policy modifies your deployment by using the `Kyverno CLI`. https://kyverno.io/docs/kyverno-cli/. You could also deploy this policy to a test system, and then restart your pods, etc.